api/proto/banyandb/measure/v1/query.proto

```diff
@@ -90,4 +90,10 @@ message QueryRequest {
   } 
   // top limit the result based on a particular field
   Top top = 9;
+  // offset is used to support pagination, together with the following limit
+  uint32 offset = 10;
```

Comment:
But what's the difference between `top` and `limit/orderBy`?

Reply:
It seems we should merge these two?

Reply:
`top` operation is based on `field` while `limit/order by` is on `tag`.  They are different.

Reply:
> `top` operation is based on `field` while `limit/order by` is on `tag`. They are different.

Oh, I see. Then which one has high priority? Or they are exclusive?

Reply:
We can apply them at the same time. `limit` returns the results based on `top`'s output.

Reply:
I will add more details in the comments.

Reply:
done

Reply:
Normally, we don't use topN mixed with limit/offset right?

Reply:
Exactly. We don't have such a use case in OAP integration.